testing-guidance/workflow-tests/editor/editor-workflow-tests.md

```diff
@@ -24,12 +29,13 @@ Saved and exported level with an interesting scene
+| Create a new level/Open an existing level                              |                                                                                                                                                                                                                               | <ul><li>Excessive load times.</li></ul>                                                                                                                                    |
+| Create several new entities                                            | <ol><li>Create some in Entity Outliner/some in Viewport.</li><li>Try duplicating/copy pasting entities to create new copies.</li><li>Drag in prefabs from Asset Browser.</li><li>Instantiate prefabs from context menu.</li><li>Undo/Redo.</li></ol>| <ul><li>Entity names.</li></ul>                                                                                                                      |
+| Add/edit various components to entities to create an interesting scene | <ol><li>As components are added to entities, position entities around Viewport using Transform/Rotate/Scale controls with snapping both enabled/disabled.</li><li>Make changes via Transform component as well.</li><li>Undo/Redo.</li></ol>| <ul><li>Component property control issues (Sliders/dropdowns/text boxes).</li><li>Component dependencies.</li><li>Rendering changes reflected in Viewport.</li><li>Viewport controls.</li></ul>|
 | Enter Game Mode                                                        |                                                                                                                                                                                                                               |                                                                                                                                                                            |
-| Save/Export level                                                      |                                                                                                                                                                                                                               | *   Asset Processor errors with level processing                                                                                                                           |
+| Save/Export level                                                      |                                                                                                                                                                                                                               | <ul><li>Asset Processor errors with level processing.</li></ul>                                                                                                            |
+---
 
 ### Workflow 2: UI Layout Customization, Docking/Window Management
```

Comment:
It would be cool to see some configuration options here for common resolution standards and windows scaling

If you use a table and linking off to those tables similar to how some of the Physics workflows reference shapes/collider types this could make such a workflow option easy to maintain understand.

Resolutions: 1080p (1920x1080), 4k (3840 x 2160), WQD (3440x1440), WQD+ (3840×1600).
Scaling: 100, 125, 150, 175, 200

I've see issues with Qt in O3DE (as well as other applications) that have had issues when combining these things or going back and forth out of full screen and windowed mode on WQD, or restarting a an app in full screen mode in WQD.

Reply:
The physics lkg workflow uses the reference sections that link to a reference list at the top of the doc.

As for the instructions list out the resolutions and resolution scalings as two lists and write the workflow to just mention using a resoltion & scaling.

You can write it in such way that during normal testing somebody cna run through 1-3 combinations but during a full regression that all listed resolutions and scaling settings are hit.

Reply:
I have added the Resolution/Scaling workflow task and did minor changes. Please let me know if there is anything else to add.